moj_auth/__init__.py

```diff
@@ -115,3 +116,7 @@ def logout(request):
 
     if hasattr(request, 'user'):
         request.user = MojAnonymousUser()
+
+
+def urljoin(base, *parts):
+    return '/'.join([s.strip('/') for s in [base] + list(parts)]) + '/'
```

Comment:
Is there a reason you're doing this rather than [`urllib.parse.urljoin`](https://docs.python.org/dev/library/urllib.parse.html#urllib.parse.urljoin)?